app/controllers/banners_controller.rb

```diff
@@ -34,6 +38,8 @@ def update
     authorize :application, :admin_or_supervisor?
 
     @banner = current_organization.banners.find(params[:id])
+    deactivate_alternate_active_banner if banner_params[:active]&.to_i == 1
```

Comment:
I think you could move the `if banner_params[:active]&.to_i == 1` to the `deactivate_alternate_active_banner` method for a little better readability.

Also, `nil.to_i = 0`, so you can get away with: `banner_params[:active].to_i == 1`

Reply:
Ah, I like this. Thanks!